blint/analysis.py

```diff
@@ -343,9 +343,10 @@ def report(src_dir, reports_dir, findings, reviews, files, fuzzables):
     if not findings and not reviews:
         LOG.info(f":white_heavy_check_mark: No issues found in {src_dir}!")
     # Try console output as html
-    html_file = Path(reports_dir) / "blint-output.html"
-    console.save_html(html_file, theme=MONOKAI)
-    LOG.info(f"HTML report written to {html_file}")
+    if findings or reviews:
```

Comment:
@ittach1 Let's just do an else here since we will know there are findings or reviews based on the if statement on line 343.

Reply:
@cerrussell Makes sense. Made the suggested changes, please review it.